### PR TITLE
V3 armature reader

### PR DESCRIPTION
**Pull request by doomsdayer**
armature reader feature.
with test and resources and projects

### CODE REVIEW

**Comment by minggo** [comment]
[ci rebuild]

**Comment by doomsdayer** [comment]
load a cocosstudio 1.x animation editor published project (.ExportJson) in a csb(cocosstudio2.0.8+ published) as a Node in cocos2d.

**Comment by doomsdayer** [comment]
I have create a PR for the resource of this feature,  but the pr is not merged until now.

**Comment by minggo** [comment]
Please paste the PR url for resource here.

File: cocos/editor-support/cocostudio/WidgetReader/ArmatureNodeReader/ArmatureNodeReader.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
+
```
Comment (minggo) [inline cocos/editor-support/cocostudio/WidgetReader/ArmatureNodeReader/ArmatureNodeReader.h:24]: Please add license in header files.

Comment (doomsdayer) [inline cocos/editor-support/cocostudio/WidgetReader/ArmatureNodeReader/ArmatureNodeReader.h:24]: The license has been added.

**Comment by minggo** [comment]
Please fix conflict.

**Comment by doomsdayer** [comment]
I have pulled the main branch and resolved the conflict this morning, now it's compiling...

**Comment by minggo** [comment]
Yep. Because many pull requests will modify project files.
So it is easy to cause conflict.

**Pull request merged**